scanner/metadata_ffmpeg.go

```diff
@@ -269,23 +269,17 @@ func (m *Metadata) parseDuration(tagName string) float32 {
 }
 
 func createProbeCommand(inputs []string) (string, []string) {
-	cmd := conf.Server.ProbeCommand
-
-	split := strings.Split(cmd, " ")
 	args := make([]string, 0)
-	first := true
-	for _, s := range split {
-		if s == "%s" {
```

Comment:
I'd prefer to keep using `%s` to be replaced by the file(s) need to be probed. This way we can have a flexible probe command, that can accept the file name in any part of the command, not only at the end.